internal/pkg/instrumentation/probe/probe.go

```diff
@@ -188,7 +188,7 @@ func (i *Base[BPFObj, BPFEvent]) loadUprobes(exec *link.Executable, td *process.
 				logFn = i.Logger.Warn
 			default:
 				// Unknown and FailureModeError.
-				return fmt.Errorf("uprobe %s package constraint (%s) not meet", up.Sym, pc.Constraints.String())
+				return fmt.Errorf("uprobe %s package constraint (%s) not met, version %v", up.Sym, pc.Constraints.String(), td.Modules[pc.Package])
```

Comment:
```suggestion
				return fmt.Errorf("uprobe %s package constraint (%s) not meet, version %v", up.Sym, pc.Constraints.String(), td.Modules[pc.Package])
```

Reply:
Hey @MrAlias, I'm happy to change it back from 'met' to 'meet', I thought it was more correct form. The package constraint is not met, I think, or maybe I misunderstand the wording.